plugins/janus_videocall.c

```diff
@@ -1358,20 +1358,15 @@ static void *janus_videocall_handler(void *data) {
 			json_t *msg_simulcast = json_object_get(msg->jsep, "simulcast");
 			if(msg_simulcast && janus_get_codec_pt(msg_sdp, "vp8") > 0) {
 				JANUS_LOG(LOG_VERB, "VideoCall callee (%s) is going to do simulcasting\n", session->username);
-				session->ssrc[0] = json_integer_value(json_object_get(msg_simulcast, "ssrc-0"));
-				session->ssrc[1] = json_integer_value(json_object_get(msg_simulcast, "ssrc-1"));
-				session->ssrc[2] = json_integer_value(json_object_get(msg_simulcast, "ssrc-2"));
+		        int rid_ext_id = -1;
+				janus_rtp_simulcasting_prepare(msg_simulcast, &rid_ext_id, session->ssrc, session->rid);
+				session->sim_context.rid_ext_id = rid_ext_id;
```

Comment:
As I pointed out, callees cannot do simulcast. It's likely the whole code above (the one already in the repo) is redundant and should be removed.

Reply:
yes, I will remove it.

Reply:
As soon as you remove tis part here, it's good to merge for me :+1:

Reply:
Note: I can get rid of the whole existing (and unneeded) simulcast check for callees myself later.